integration-tests/easy-rag/src/test/resources/__snapshots__/EasyRagCodestartTest/testContent/src_main_java_ilove_quark_us_Bot.java

```diff
@@ -0,0 +1,34 @@
+package ilove.quark.us;
+
+import dev.langchain4j.service.SystemMessage;
+import dev.langchain4j.service.UserMessage;
+import io.quarkiverse.langchain4j.RegisterAiService;
+import jakarta.enterprise.context.SessionScoped;
+
+/**
+ * This is a sample Bot, it is configured to ingest the 'easy-rag-catalog/'.
+ * You can @Inject this Bot in your Rest resource
+ *
+ * <code>
+ *     @Inject
+ *     Bot bot;
+ *
+ *     @POST
+ *     @Produces(MediaType.TEXT_PLAIN)
+ *     public String chat(String q) {
+ *         return bot.chat(q);
+ *     }
+ * </code>
+ */
+@RegisterAiService // no need to declare a retrieval augmentor here, it is automatically generated and discovered
+@SessionScoped
```

Comment:
Also, let's not use `@SessionScoped` by default, so just remove this

Reply:
I think @maxandersen added this on purpose it's in the readme

Reply:
In any case, it's wrong. It only makes sense when used with `quarkus-websockets-next`

Reply:
```
To keep a chat history, you need a session. You can use `http --session mychat :8080/hello q='what products are available?'` to query the bot,
and then follow up with `http --session mychat :8080/hello q='what is #2 about ?'` and it will continue the conversation.
```

We could change it to:

To keep a chat history, you need `@SessionScoped` on the Bot. You can use `http --session mychat :8080/hello q='what products are available?'` to query the bot with a session,
and then follow up with `http --session mychat :8080/hello q='what is #2 about ?'` and it will continue the conversation.

Reply:
That is still not correct really, because there is chat history for the default (`@RequestScoped`). 

`@SessionScoped` will **only** work with `quarkus-websockets-next` and even if that is around, it only makes sense to use it as the scope of services injected into the WebSocket handlers.

Reply:
Removed

Reply:
Thanks!